include/usb/usb_common.h

```diff
@@ -104,6 +104,7 @@
 	   (USB_CONFIGURATION_ATTRIBUTES_REMOTE_WAKEUP), (0))))
 
 /* Classes */
+#define AUDIO_CLASS			0x01
```

Comment:
At some point we may rename this and others to `USB_CLASS_AUDIO`, etc

Reply:
I've just done it to suit to present defines. I think I will do something about that to resolve #20236 just after I make all corrections to this pr and it will be merged.